packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -1224,3 +1224,15 @@ new rds.DatabaseCluster(this, 'DatabaseCluster', {
   preferredMaintenanceWindow: 'Sat:22:15-Sat:22:45',
 });
 ```
+You can also use writer or readers props like:
```

Comment:
nit: "You can also set the preferred maintenance window via reader and writer props:"

Reply:
Sounds good thanks